package.json

```diff
@@ -11,9 +11,10 @@
     "unit": "tap -J test/*.test.js test/*/*.test.js",
     "unit:report": "tap -J test/*.test.js test/*/*.test.js --cov --coverage-report=html --coverage-report=cobertura | tee out.tap",
     "unit:junit": "tap-mocha-reporter xunit < out.tap > test/junit-testresults.xml",
+    "unit:compile": "tsc --project ./test/classBasedPlugins/tsconfig.json",
     "typescript": "tsc --project ./test/types/tsconfig.json",
-    "test:report": "npm run lint && npm run unit:report && npm run typescript",
-    "test": "npm run lint && npm run unit && npm run typescript",
+    "test:report": "npm run lint && npm run unit:compile && npm run unit:report && npm run typescript",
```

Comment:
I'm concerned with how these .ts tests will affect the coverage report. Can you verify the current test coverage reporter is accurately reporting that the new API code is being covered by these tests?

Reply:
The coverage is being reported correctly if the test have been compiled.
However I forgot compile the typescript tests in `coverage` and `coveralls` tasks. I'll change that.